ios/NewExpensifyShare/ShareViewController.swift

```diff
@@ -0,0 +1,260 @@
+//
+//  ShareViewController.swift
+//  RNShareMenu
+
+import Foundation
```

Comment:
This looks like the file was copied into the NewExpensifyShare target instead of linking to the copy inside react-native-share-menu. (Note in instructions: "On the pop-up, select node_modules/react-native-share-menu/ios/ShareViewController.swift. **Make sure Copy items if needed is NOT selected**"). If I'm wrong let me know but I'm pretty sure the diff should only show this way if it was copied and not just referencing node_modules?

This copying method is what the old prototype branch was doing, so the confusion would be understandable. This was because, for the share extension, we needed to customize this more and it seemed like copying the file was the best way.

But the nice thing about just referencing the node_modules copy is that we can automatically upgrade the controller in future releases of RNSM, instead of giving people native code diffs to manually apply.